src/assertions.jl

```diff
@@ -7,3 +7,11 @@ function assert_continuous(table)
   types = schema(table).scitypes
   @assert all(T <: Continuous for T in types) "columns must hold continuous variables"
 end
+
+function assert_continuous_or_missing(table)
+  cols = Tables.columns(table)
+  for col in cols
+      type = scitype(col)
+      @assert type <: AbstractVector{<:Union{Missing, Continuous}} "columns must hold continuous variables"
```

Comment:
Can you elaborate on why we are not using the same pattern of the assert_continuous function with T <: Union{<:Continuous,Missing} ?

Reply:
I simply implemented the suggestion made in #27 .  I also agree that `T <: Union{<:Continuous,Missing}` is a more elegant solution.